src/components/inner-block-slider/inner-block-single-display.js

```diff
@@ -38,7 +38,7 @@ function InnerBlocksDisplaySingle( {
 			allowedBlocks,
 			orientation: 'horizontal',
 			renderAppender,
-			template,
+			...( template ? { template } : null ),
```

Comment:
Can you spread null?

Reply:
You can! It comes out to `{}`. 
<img width="189" alt="image" src="https://github.com/user-attachments/assets/3466b7ed-0134-4f89-8608-b0c2471e91af">
`undefined` works too.